packages/example/src/client.ts

```diff
@@ -15,20 +15,29 @@
 import { createPromiseClient } from "@bufbuild/connect";
 import { createConnectTransport } from "@bufbuild/connect-node";
 import { ElizaService } from "./gen/eliza_connect.js";
-import { stdin, stdout } from "process";
+import { stdin, stdout, env } from "process";
 import * as readline from "node:readline/promises";
 
 const rl = readline.createInterface(stdin, stdout);
 
-void (async () => {
-  const transport = createConnectTransport({
-    httpVersion: "2",
-    baseUrl: "https://localhost:8080",
-    nodeOptions: {
-      rejectUnauthorized: false,
-    },
-  });
+let rejectUnauthorized = true;
+// prettier-ignore
+if (env.NODE_EXTRA_CA_CERTS == undefined) {
+  rl.write("! Looks like you did not let Node.js know about your certificate authority for \n");
+  rl.write("! local development yet. This is fine, we'll ignore TLS errors in this example \n");
+  rl.write("! client. \n\n");
+  rejectUnauthorized = false;
+}
```

Comment:
Of course Node.js does not pick up OS CAs - that would be too easy 😆 With this if-block, you can still run the Node.js server and client without using `mkcert` at all.